src/liballoc/tests/btree/map.rs

```diff
@@ -758,10 +760,7 @@ fn test_bad_zst() {
 #[test]
 fn test_clone() {
     let mut map = BTreeMap::new();
-    #[cfg(not(miri))] // Miri is too slow
-    let size = 100;
-    #[cfg(miri)]
-    let size = 30;
+    let size = 12; // to obtain height 2 tree (having edges to leaf nodes)
     assert_eq!(map.len(), 0);
 
     for i in 0..size {
```

Comment:
In some earlier version this became `=size`... so is size 11 or size 12 the first height-2-size?

Reply:
12 elements are needed to get height 2 (though if you take out the right element, you can have a height 2 tree with 11 elements). In `test_clone_from`, I put the `=size` because there `i` is the size being tested and `size` is really `max_size_tested`.

Reply:
Ah, but here it's just "go 12 times around the loop"? Makes sense.